icgc.bionimbus.org/portal/gitops.json

```diff
@@ -104,13 +104,11 @@
         "alt": "Center for Translational Data Science at the University of Chicago"
       },
       {
-        "src": "/src/img/sponsors/nhlbi.png",
-        "href": "https://www.nhlbi.nih.gov",
-        "alt": "NHLBI"
+        "src": "/src/img/sponsors/icgc.png",
```

Comment:
should be ICGC.png? uppercased...

Reply:
That sounds smart. Changed per file name being uppercase.